Registry/AssetSystemOptions.assetbuilder.setreg

```diff
@@ -0,0 +1,12 @@
+{
+    "O3DE":
+    {
+        "AssetSystem":
+        {
+            // Turn off the ability for the asset system to send synchronous blocking requests,
+            // for asset builders.  This is becuase they themselves produce assets, and if they are allowed
+            // to issue blocking requests, this can lead to deadlocks.
+            "AllowSyncRequests" : false
+        }
```

Comment:
Authors note.  The Setreg system uses the application name ("assetbuilder") to filter out this file to anything NOT assetbuilder.  All setreg work this way (if they have a `.tag.` in them, they only load in things which set that tag).